src/main/java/teammates/ui/controller/InstructorCourseEnrollPageData.java

```diff
@@ -6,14 +6,28 @@
  * PageData: this is page data for 'Enroll' page for a course of an instructor
  */
 public class InstructorCourseEnrollPageData extends PageData {
+    private String courseId;
+    private String enrollStudents;
     
     public InstructorCourseEnrollPageData(AccountAttributes account) {
         super(account);
         enrollStudents = "";
     }
-
-    public String courseId;
     
-    public String enrollStudents;
-
+    public void init(String courseId, String enrollStudents) {
+        this.courseId = courseId;
+        this.enrollStudents = enrollStudents;
```

Comment:
This can be combine with the constructor as well.
The `enrollStudents = ""` in the constructor right now can be removed.